content/english/datastruct/2D Arrays/example.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Example"
+draft: false
+weight: 3
+---
+![image](../../img/2dexample.png)
+<link rel="stylesheet" href="../../style.css">
+<hr>
+
+2D arrays work very similarly to regular arrays and many of the ideas or methods that we use on arrays we can apply on multidimensional arrays as well. As a quick reminder, here is how to initiliaze a 2D array and access an element.
+
+```js javascript
+public static void main(String[] args) 
+{ 
+  
+    int[][] arr = new int[10][20]; 
+    arr[0][0] = 1; 
+    System.out.println(arr[0][0]); 
+} 
+    //outputs 1
+```
+
+The only hard part that might not make a whole lot of sense at first is iterating through a 2D array, but this is quite simple as well. Instead of a single for loop, we will use two for loops instead. Here is a quick example to give you an idea. 
```

Comment:
```suggestion
The only hard part that might not make a whole lot of sense at first is iterating through a 2D array, but this is quite simple as well. Instead of a single `for` loop, we will use two `for` loops instead. Here is a quick example to give you an idea. 
```